csrc/executor.h

```diff
@@ -276,17 +278,19 @@ class TORCH_CUDA_CU_API FusionExecutor : public NonCopyable {
       bool align_padding = false,
       uint64_t total = 0);
 
-  // return a pair of vector of tensors, where tensors in the first vector are
-  // not initialized, while the second vector contains zero-initiliazed tensors
-  GlobalBuffers allocGlobalVals(ExpressionEvaluator& expr_eval);
+  //! Return information necessay for allocating intermediate tensors,
+  //! including temporary work buffers as well as intermediate
+  //! global-memory tensors
+  std::vector<GlobalBufferInfo> getIntermediateBufferInfo(
```

Comment:
I wanted to use different names for outputs and temporary tensors that are just used inside a kernel since they are managed and allocated differently, but both of them are global-memory tensors, thus `GlobalBuffer`.